coderd/audit.go

```diff
@@ -182,6 +182,7 @@ func (api *API) convertAuditLogs(ctx context.Context, dblogs []database.GetAudit
 type AdditionalFields struct {
 	WorkspaceName string
 	BuildNumber   string
+	BuildReason   string
 }
```

Comment:
Should these have json tags? Since they are stored as json? I recognize this would break backwards compatibility :cry: 

Also can this be the `database.BuildReason` type?

Reply:
Added! Thanks for the review!